### PR TITLE
feat: skip p2p-circuit addresses

### PR DESCRIPTION
**Pull request by dryajov**
Ignore `/p2p-circuit` addresses

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -68,7 +68,9 @@ class TCP {
       multiaddrs = [multiaddrs]
     }
     return multiaddrs.filter((ma) => {
-      if (includes(ma.protoNames(), 'ipfs')) {
+      if (includes(ma.protoNames(), 'p2p-circuit')) {
+        return false
+      } else if (includes(ma.protoNames(), 'ipfs')) {
```
Comment (diasdavid) [inline src/index.js]: Use `mafmt` for this instead?

Comment (dryajov) [inline src/index.js]: I do, but decapsulated will strip the `ipfs` part along with `p2p-circuit`, so mafmt wont match it. Mafmt does have all the logic to do the matches correctly tho.

**Comment by dryajov** [comment]
not needed anymore, going to be handled by mafmt

**Comment by dryajov** [comment]
@diasdavid can we get this reopened, as per our discussion earlier today? 👍

**Pull request closed without merging**